psydac/ddm/cart.py

```diff
@@ -1124,110 +1138,142 @@ def __init__(self, npts, pads, periods, comm, shifts, axes, exts, ranks_in_topo,
     #---------------------------------------------------------------------------
     @property
     def ndims( self ):
+        """Number of dimension."""
         return self._ndims
 
     @property
     def npts_minus( self ):
+        """Number of points in the minus side of an interface."""
         return self._npts_minus
 
     @property
     def npts_plus( self ):
+        """Number of points in the plus side of an interface."""
         return self._npts_plus
 
     @property
     def pads_minus( self ):
+        """ padding in the minus side of an interface."""
         return self._pads_minus
 
     @property
     def pads_plus( self ):
+        """ padding in the plus side of an interface."""
         return self._pads_plus
 
     @property
     def periods_minus( self ):
+        """ Periodicity in the minus side of an interface."""
         return self._periods_minus
 
     @property
     def periods_plus( self ):
+        """ Periodicity in the plus side of an interface."""
         return self._periods_plus
 
     @property
     def shifts_minus( self ):
+        """ The shift values in the minus side of an interface."""
         return self._shifts_minus
 
     @property
     def shifts_plus( self ):
+        """ The shift values in the plus side of an interface."""
         return self._shifts_plus
 
     @property
     def ext_minus( self ):
+        """ the extremity  of the boundary on the minus side of an interface."""
         return self._ext_minus
 
     @property
     def ext_plus( self ):
+        """ the extremity of the boundary on the plus side of an interface."""
         return self._ext_plus
 
     @property
     def root_rank_minus( self ):
+        """ The root rank of the communicator defined in the minus patch."""
         return self._root_rank_minus
 
     @property
     def root_rank_plus( self ):
+        """ The root rank of the communicator defined in the plus patch."""
         return self._root_rank_plus
 
     @property
     def ranks_in_topo_minus( self ):
+        """ the ranks in the cartesian topology of defined on the minus patch."""
         return self._ranks_in_topo_minus
 
+    @property
+    def ranks_in_topo_plus( self ):
+        """ the ranks in the cartesian topology of defined on the plus patch."""
+        return self._ranks_in_topo_plus
```

Comment:
I cannot understand these docstrings... Maybe "of" is out of place? Do these properties return a list?

Reply:
yes, "of" is out of place, and yes they return a numpy array, they map the ranks in the communicator to ranks in the cartesian topology

Reply:
Then I suggest that you use a docstring similar to one you wrote before:
"Array that maps the ranks in the intra-communicator on the plus(minus) patch to their rank in the corresponding Cartesian topology."

Reply:
done in [5c14c72](https://github.com/pyccel/psydac/pull/213/commits/5c14c7284c7e6c151b6d328777bb6f90a4d681b6)